packages/suite/src/views/settings/index.tsx

```diff
@@ -199,6 +211,58 @@ const Settings = ({
                 )}
             </Section>
 
+            <Section title={<Translation id="TR_TOR" />}>
+                <SectionItem>
+                    <TextColumn
+                        title={<Translation id="TR_TOR_TITLE" />}
+                        description={
+                            <Translation
+                                id="TR_TOR_DESCRIPTION"
+                                values={{
+                                    lineBreak: <br />,
+                                }}
+                            />
+                        }
+                        learnMore="https://www.torproject.org/"
+                    />
+                    <ActionColumn>
+                        <Switch
+                            data-test="@settings/metadata-switch"
```

Comment:
this line is breaking e2e tests. It would be best to remove data-test attr at the moment.

Reply:
d61f666